docs/community/contribute/compliance.md

```diff
@@ -0,0 +1,151 @@
+# Contribute Compliance Checks in Trivy
+
+Trivy supports several different compliance checks. The details on compliance scanning with Trivy are provided in the [compliance documentation](../../docs/compliance/compliance.md).
+All of the Compliance Checks currently available in Trivy can be found in the `trivy-policies/specs/compliance/` directory ([Link](https://github.com/aquasecurity/trivy-policies/tree/main/specs/compliance)).
+
+New checks are based on the custom compliance report detailed in the [main documentation.](../../docs/compliance/compliance/#custom-compliance)
+If you would like to create your custom compliance report, please reference the information in the main documentation. This section details how community members can contribute new Compliance Specs to Trivy.
+
+All compliance specs in Trivy are based on the formal compliance reports such as CIS Benchmarks.
+
+## Contributing new Compliance Specs
+
+Compliance specs can be based on new compliance reports becoming available, or identifying missing compliance specs that Trivy users would like to access. 
+
+### Create a new Compliance Spec
+
+The existing compliance specs in Trivy are located under the `trivy-policies/specs/compliance/` directory ([Link](https://github.com/aquasecurity/trivy-policies/tree/main/specs/compliance)).
+
+Create a new file under `trivy-policies/specs/compliance/` and name the file in the format of "provider-resource-spectype-version.yaml". For example, the file name for AWS CIS Benchmarks for EKS version 1.4 is: `aws-eks-cis-1.4.yaml`.
+
+### Minimum spec structure
```

Comment:
IMO there's too much duplication with the general compliance documentation, this should be about how to upstream a compliance, not the complete guide to write a compliance.

Reply:
Does this relate to line 19 or to the minimum spec structure?

Reply:
I have moved some to the normal compliance documentation but I am thinking about the following instead:
- divide the compliance docs into the following two separate sections in the table of content: built-in compliance and custom compliance. The custom compliance would then also detail how to contribute the custom compliance check to Trivy

Reply:
> Does this relate to line 19 or to the minimum spec structure?

Minimum spec structure (and what follows)

Reply:
> table of content: built-in compliance and custom compliance

not sure I understand how your suggestion is different than existing?
<img width="149" alt="image" src="https://github.com/aquasecurity/trivy/assets/1161307/d803d18f-459e-4b2c-b9cf-c729f1083e24">